source/_integrations/tplink_omada.markdown

```diff
@@ -37,26 +37,29 @@ TP-Link Omada Controller:
 
 - OC200
 - OC300
-- Software Controller.
+- Software Controller
 
 Controller versions 5.1.0 and later are supported.
 
 ## Supported Omada devices
 
 ### Network Switches
 
-- Support for enabling/disabling Power over Ethernet on a per-port basis.
-- Firmware Update entities.
+- Support for enabling/disabling Power over Ethernet on a per-port basis
+- Firmware Update
 
 ### Access Points
 
-- Firmware Update entities.
+- Firmware Update
 
 ### Internet Gateways
 
-- Firmware Update entities.
-- WAN/LAN Port connectivity sensor entities.
+- WAN/LAN Port connectivity sensors
+- WAN Port Online detection sensors
+- WAN Port Connect/Disconnect switches
+- LAN Port PoE activity sensor
+- Firmware Update entities
 
 ## Device trackers
 
-- Not currently supported.
+- Not currently supported
```

Comment:
If it's not currently supported, wouldn't it be better to remove it?

Reply:
Fair point. Removed.